### ISSUE TITLE
Investigate packaging process excluding Sass files

### ISSUE THREAD

**Issue by Dananji** [samvera-labs/iiif-react-media-player#20]
### Description

When packaging iiif-react-media-player, the `.scss` files do not get included. 

### Done Looks Like

When packaged the `dist` folder includes the stylesheets (.scss files) from `src` directory

**Comment by Dananji** [comment]
This ticket refers to the packaging process of the previous [react-iiif-media-player](https://github.com/avalonmediasystem/react-iiif-media-player) repository.

I tried a couple of different webpack configs trying to make this work and then talked to Adam to get help on it. And we discussed and decided that trying to maintain this is not worth the effort since we have a new player with similar functionality in the component library we developed.

Suggestion: archive the previous repository and change the README.md file to reflect that, the component was developed as a prototype and now the new working player lives in the new component library (link to the new component library). This will be shown in the npm registry's page for the previous component, redirecting the traffic to the new component library.
